sympy/polys/ring_series.py

```diff
@@ -1140,6 +1140,8 @@ def rs_exp(p, x, prec):
     n = 1
     k = 1
     c = []
+    if not isinstance(prec, int):
+        prec = ceiling(prec)
     for k in range(prec):
```

Comment:
Can we simply do `for k in range(ceiling(k)):`?

Reply:
I mean `ceiling(prec)` ofcourse, not `ceiling(k)`.

Reply:
Yes. I just checked for the same. It works.